index.html

```diff
@@ -67,64 +60,131 @@ <h2>
       <div>
         <!-- Introduction -->
         <section>
+            <h1>
+              Woof Woof
+            </h1>
         	<div>
-
-        		<h1>
-        			Woof Woof
-        		</h1>
         		<p>
         		 Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Blandit cursus risus at ultrices mi. Volutpat odio facilisis mauris sit amet massa vitae tortor condimentum. Auctor neque vitae tempus quam.
         		 </p>
-
-        	<a href="media.html"><button>Go To The Good Boys</button></a>
-
+          <!-- button -->
+        	<a href="media.html"><div>Go To The Good Boys</div></a>
```

Comment:
no need to wrap the text in a div tag. is some cases, its better to use a span, but the <a> tag should be enough.